### ISSUE TITLE
JAWS and NVDA form fields dialogue do not show autocomplete clearly

### ISSUE THREAD

**Issue by nickcolley** [alphagov/accessible-autocomplete#344]
This issue is from a May 2019 external accessibility audit report.

**WCAG Reference**:
- [1.3.1 Info and Relationships (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
**Issue ID**: DAC_4

## Summary
When trying to navigate to the autocomplete using form fields dialogue it is not clear.

> While in the form fields dialogue menu (‘insert+f5’), the form element announced only as “No search results.” It was therefore challenging to identify which form element should be selected and what the field related to.
>
> While using NVDA, I found that the field was unlabelled which could mean that users are not aware of what they are interacting with. While using VoiceOver for iPhone and Mac while in context, the autocomplete reads information relating to the number of results, but with the form fields rotor, two separate items are located relating to the results and search information which might not be clear to users.

## Screen Shot
![](https://user-images.githubusercontent.com/2445413/60200745-9b281380-983e-11e9-8226-74b75ae5b52a.png)

**Comment by nickcolley** [comment]
This may be resolved by HMRC's work https://github.com/alphagov/accessible-autocomplete/pull/339

> Form fields are now visible in overview (NVDA Insert + F7 , rotor in VoiceOver)
> 
> (Related to the above) Form fields can now be selected directly using relevant form navigation key (e.g. key 'F' in NVDA)

**Comment by nickcolley** [comment]
Will be in the 2.0.0 release :)

**Comment by nickcolley** [comment]
Confirmed as fixed in a re-test by DAC on 7th October 2019
